Content.Shared/Storage/EntitySystems/SharedStorageSystem.cs

```diff
@@ -463,7 +463,7 @@ private void AfterInteract(EntityUid uid, StorageComponent storageComp, AfterInt
                 return;
             }
 
-            if (_xformQuery.TryGetComponent(uid, out var transformOwner) && TryComp<TransformComponent>(target, out var transformEnt))
+            if (TryComp(uid, out TransformComponent? transformOwner) && TryComp(target, out TransformComponent? transformEnt))
```

Comment:
Not using xformQuery anymore?

Reply:
The transform & metadata have non-generic variants that automatically use xform & metadata queries. i.e.,
```cs
protected bool TryComp(EntityUid uid, [NotNullWhen(true)] out TransformComponent? comp)
{
    return EntityManager.TransformQuery.TryGetComponent(uid, out comp);
}
```
The `Transform(EntityUid uid)` `Metadata(EntityUid uid) methods also use them, so there generally shouldn't be a need to use xform & metadata queries